components/display-messages/notifications.qmd

```diff
@@ -36,131 +36,166 @@ listing:
-          Define a function within the `server` function that builds your table and uses `render.DataTable()` to return the result. Where`render.DataTable()`will package the table as an HTML data table for display. If your function calls reactive values, Shiny will update your table whenever those values change, in a [reactive fashion]().
-          
-          To notify Shiny that your function returns a reactive table to display, decorate the function with two decorators:
-          
-           <ul>
-             <li>`@output`</li>
-             <li>`@render.data_frame()`</li>
-           </ul>
+        ```
+        @reactive.Effect
+        @reactive.event(input.show)
+        def _():
+            ui.notification_show("You've been notified.")
+        ```
 
-          </li>
+        You can call [`ui.remove_notification()`](TODO ADD LINK TO UI REMOVE NOTIFICATION) to remove a notification programatically, but usually app developers will let notifications expire on their own. Also, notifications come by default with a button that the user can click to close the notification prematurely.
 
-          <li>Pass `ui.output_data_frame()` the name of the function that builds the DataTable, as a character string, to indicate which DataTable should be displayed in the div. 
-          </li>
-        </ol>
+        ## Duration
 
-      variations:
-        - title: Filterable Table
-          description: An optional place for a description here that goes about this long but then (oh my gosh!) it's now going even longer to show what the line-height looks like.
-          preview: https://shinylive.io/py/app/#h=0&code=NobwRAdghgtgpmAXGKAHVA6VBPMAaMAYwHsIAXOcpMAMwCdiYACVKAG3jtUoHMBXAJYQAzkwExUxOmSZtiUACYB9bhH5DhAHQj1GTYQAsh2MRKkyAgujxM6lBXDo3hcYcIGkbg7dtXqRTAC8svLKfoIiABQAlD7Q6EqCQUyCWFA8cEo0bIIKkdpMKQIYxHxkqGVKClBkUFl0sHD5YOEaVTSaYLEQ3doONPqOAG6OkUIVZDal5WXOru6k0YgFhUwAAtMTK4VrdhAOdBjVtfWN20z9LLwRwu0xyxCrT7ZwZHx0j3sHGAAiNVAAFSgACM2E1zs9Vq0RHgIZCmDQBGwKHRRMEAXQ+HA4YVevFUMkrKhImhUIkBHM6CM6NEwABfAC6QA
-          code: |
-            from palmerpenguins import load_penguins
-            from shiny import App, render, session, ui
+        By default, Shiny notifications will disappear after five seconds. To change how long a notification appears for, set the `duration` argument of `ui.show_notification()` to an integer number of seconds. Set `duration` to `None` to have the notification appear until the user closes it. 
+
+        ## Type
 
-            penguins = load_penguins()
+        Shiny notifications come in three types: messages, warnings and errors. To set the type of a notification, use the `type` argument of `ui.show_notification()`.
 
+        ## See Also
+
+        [Modal messages](TO DO ADD LINK TO MODAL DETAIL PAGE) provide a similar, but alternative way to display information to the user.
+
+      variations:
+        - title: Warning notification
+          description: Set `type="warning"` to create a warning colored notification.
```

Comment:
```suggestion
          description: Set `type="warning"` to create a warning-colored (yellow) notification.
```

Reply:
Thanks. I'm committing the hyphens, but leaving the colors out. It makes so much sense to include the color, but I know that with the bslib theming systems, different themes swap out the warning and error colors.